RELEASES.md

```diff
@@ -1,53 +1,72 @@
     - `MyContract` will still be the storage struct.
       However, it can now additionally be used as static dependency in other smart contracts.
       Static dependencies can be envisioned as being directly embedded into a smart contract.
     - `MyContractRef` is pretty much the same of what we had gotten with the old `ink-as-dependency`.
       It is a typed thin-wrapper around an `AccountId` that is mirroring the ink! smart contract's API
       and implemented traits.
-- ink! trait definitions via `#[ink::trait_definition]`:
+- ink! trait definitions via `#[ink::trait_definition]` ‒ [#665](https://github.com/paritytech/ink/pull/665).
     - ink! trait definitions no longer can define trait constructors.
     - ink! trait implementations now inherit `selector` and `payable` properties for trait messages.
         - Now explicitly setting `selector` or `payable` property for an implemented ink! trait method
           will only act as a guard that the set property is in fact the same as defined by the ink!
           trait definition.
-- Improved some ink! specific compile errors:
-    - For example, when using ink! messages and constructors that have inputs or
-      outputs that cannot be encoded or decoded using the SCALE codec.
-- Simplified selector computation for ink! trait methods.
+- Improved some ink! specific compile errors ‒ [#665](https://github.com/paritytech/ink/pull/665).
+    - For example, when using ink! messages and constructors which have inputs (or
+      outputs) that cannot be encoded (or decoded) using the SCALE codec.
+- Simplified selector computation for ink! trait methods ‒ [#665](https://github.com/paritytech/ink/pull/665).
     - Now selectors are encoded as `blake2b({namespace}::{trait_identifier}::{message_identifier})[0..4]`.
       If no `namespace` is set for the ink! trait definition then the formula is
       `blake2b({trait_identifier}::{message_identifier})[0..4]`.
       Where `trait_identifier` and `message_identifier` both refer to the identifiers of the ink! trait
       definition and ink! trait message respectively.
+- We switched to Rust edition 2021 ‒ [#977](https://github.com/paritytech/ink/pull/977).
+- Update chain extension example to show argument passing ‒ [#1029](https://github.com/paritytech/ink/pull/1029).
 
 ## Fixed
-- Contracts that are compiled as root (the default) now properly revert the transaction if a message
-  returned `Result::Err`.
-    - This does not apply to ink! smart contracts that are used as dependencies. Therefore it is still possible to match against a result return type for a called dependency.
+- Contracts now revert the transaction if an ink! message returns `Result::Err` ‒ [#975](https://github.com/paritytech/ink/pull/975), [#998](https://github.com/paritytech/ink/pull/998).
+    - **Note:** This does not apply to ink! smart contracts which are used as dependencies of a contract!<br/>
```

Comment:
This was a bug and has been fixed. No longer holds true.

Reply:
So if the `adder` in our `delegator` example hypothetically would return `Result::Err` the entire transaction is reverted? So no way to match on `Result` in the calling contract?

Reply:
Not the entire transaction is reverted but the state transition of the Adder contract call.